core/variant/variant.h

```diff
@@ -751,7 +751,8 @@ class Variant {
 	uint32_t hash() const;
 	uint32_t recursive_hash(int recursion_count) const;
 
-	bool hash_compare(const Variant &p_variant, int recursion_count = 0) const;
+	// By default, performs a semantic comparison. Otherwise, numeric/binary comparison (if appropriate)
```

Comment:
```suggestion
	// By default, performs a semantic comparison. Otherwise, numeric/binary comparison (if appropriate).
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).

Reply:
Added